### PR TITLE
Remove of non required variable

### PR DESCRIPTION
**Pull request by mitogh**
Return the result of the call to `deleteEvent` on `Tribe__Events__API` as a variable is not really required or used during
the execution of this section of code.

### CODE REVIEW

File: src/functions/advanced-functions/event.php [hunk 1]
```diff
@@ -136,9 +136,7 @@ function tribe_update_event( $postId, $args ) {
 	 * @return bool false if delete failed.
 	 */
 	function tribe_delete_event( $postId, $force_delete = false ) {
-		$success = Tribe__Events__API::deleteEvent( $postId, $force_delete );
-
-		return $success;
+		return Tribe__Events__API::deleteEvent( $postId, $force_delete );
```
Comment (tr1b0t) [inline src/functions/advanced-functions/event.php:139]: :no_entry_sign: Variable "$postId" is not in valid snake_case format, try "$post_id"

<sub>hash: 2171c9e6bdcee442d9d9e9cd9b91027e</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)

**Review by lucatume — COMMENTED** [review]
The fix is fine, I'm waiting on the tests to pass.

File: src/Tribe/Importer/File_Importer_Events.php [hunk 2]
```diff
@@ -512,7 +512,10 @@ private function get_currency_position( array $record ) {
 	 * @return string
 	 */
 	private function get_post_excerpt( $event_id, $import_excerpt ) {
-		_deprecated_function( __METHOD__, 5.1.6', '$this->get_post_text_field( $event_id, $record, "event_excerpt", "post_excerpt" )' );
+		_deprecated_function( __METHOD__,
```
Comment (tr1b0t) [inline src/Tribe/Importer/File_Importer_Events.php:515]: :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 989ab547c1202381d133e6eb49de0d33</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)

**Review by lucatume — APPROVED** [review]

**Pull request merged**